### PR TITLE
LPS-150931 Web Content Translation Page does not display repeatable Fields in Workflow's Notification view

### PR DESCRIPTION
**Pull request by IstvanD**
Dear Team,

Could you please review this pull request?

I have sent a pull request to the Zulu Team as well, who checked and approved it on the Workflow side.
[They suggested letting it be checked by the team managing the Translations as well](https://github.com/liferay-zulu/liferay-portal/pull/97#issuecomment-1095206567).

Thank you and best regards,
István

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1307401_3065

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ViewTranslationDisplayContext.java [hunk 1]
```diff
@@ -132,14 +131,14 @@ public List<String> getStringValues(InfoField infoField, Locale locale) {
...
-				infoFieldValue.getValue(locale))
-		).collect(
-			Collectors.toList()
-		);
+		for (InfoFieldValue<Object> infoFieldValue : infoFieldValues) {
+			stringValues.add(
+				GetterUtil.getString(infoFieldValue.getValue(locale)));
+		}
+
+		return stringValues;
```
Comment (adolfopa) [inline modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ViewTranslationDisplayContext.java:141]: Can you use a regular loop here? See https://issues.liferay.com/browse/LPS-150112

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by liuxingye** [comment]
ci:test:relevant

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**